examples/data/head.nbest.en

```diff
@@ -0,0 +1,18 @@
+0 ||| can you do it in one day ?
```

Comment:
It'd be nice if we could modify this a little bit to show that these are in fact different hypotheses for the same source sentence.